app/controllers/foreman_kubevirt/concerns/api/compute_resources_controller_extensions.rb

```diff
@@ -0,0 +1,34 @@
+module ForemanKubevirt
+ module Concerns
+    module Api::ComputeResourcesControllerExtensions
+      module ApiPieExtensions
+        extend ::Apipie::DSL::Concern
+        update_api(:create, :update) do
+          param :compute_resource, Hash do
+            param :token, String, :desc => N_("Token for Kubevirt only")
```

Comment:
I think that instead of `Kubevirt` we should use the same capitalization as the friendly name: "KubeVirt".
Is there such convention on Foreman ?

Reply:
@masayag Done